modules/ROOT/pages/websockets/websocket-connection-management-components.adoc

```diff
@@ -0,0 +1,65 @@
+= About Websocket Connection Management Components
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: connectors, ws, wss, websocket
+
+== About Connection Management Components
+
+These components provide tools for connection management for both server and client side operations. All of these operations are available for both Inbound and Outbound sockets.
+
+=== On Socket Closed
+
+This component's owner flow will be triggered each time a connection is closed. This is useful to perform cleanup operations such as logging or updating the application's state.
+
+
+[source,xml,linenums]
+----
+<flow name="onConnectionClosed">
+    <websocket:on-connection-closed ​path​=​"/ws/*" ​config-ref​=​"ws"​ />
+    <logger level="INFO" message="No more messages please: you shall not pass." />
+</flow>
+----
+
+Once this flow is invoked, no more messages can be processed because the socket has already been closed.
+
+=== Subscribe Groups
```

Comment:
for this to be relevant to the reader, give a message segregation example use case.